### DISCUSSION TITLE
Should I close connections manually?

### DISCUSSION THREAD

**Discussion by user72356** [supabase/supabase#11947] [Questions]
I'm using serverless functions at Netlify and within them I'm querying a Supabase DB using the [porsager/postgres](https://github.com/porsager/postgres) client, which has its own internal pool (I think). I use the 6543 port, so I assume connections are pooled by the Supa pgBouncer server too. 

I'm not sure whether I should close the connections manually within the function code, or if they are closed automatically when the function is done running.

Additionally, since the connections are taken from a pool, I shouldn't need to close them since open connections will be reused by the next caller?

All of this together is a bit confusing to me. Any best practices out there? Thank you!

**Comment by steve-chavez** [comment]


**Reply by user72356** [reply]
I'm using the postgres.js client not postgREST (https://github.com/porsager/postgres)

**Reply by steve-chavez** [reply]
Sorry, disregard the above then.
